tests/src/test/scala/com/nvidia/spark/rapids/GpuBatchUtilsSuite.scala

```diff
@@ -25,6 +25,7 @@ import org.scalatest.funsuite.AnyFunSuite
 import org.apache.spark.sql.Row
```

Comment:
I had to make changes to this file because of the way InternalRow accesses Arrays, Maps and Structs when using `BoundReference`